tests/semantic/hex1/hex1.dl

```diff
@@ -5,15 +5,14 @@
 // - <souffle root>/licenses/SOUFFLE-UPL.txt
 
 // Test allowed range of hexadecimal integers
-.decl Hexadecimal (n:number)
+.decl Hexadecimal (n:unsigned)
 .output Hexadecimal
 
 Hexadecimal(0x1).
+Hexadecimal(0xFFFFFFFF).
 
 #if RAM_DOMAIN_SIZE == 32
-Hexadecimal(0x00000000).
-Hexadecimal(0xFFFFFFFF).
+Hexadecimal(0xFFFFFFFF + 1).
```

Comment:
As for binary, the overflow tests aren't needed. Previously it was required just to represent the binary number.